.github/workflows/percy-workflow.yml

```diff
@@ -8,19 +8,21 @@ on:
       - master
     paths:
       - '**.js'
-      - '**.json'
+      - '!src/site/_data/contributors.js'
+      - '!src/site/_data/countries.js'
+      - '!src/site/_data/postTags.js'
+      - 'package.json'
       - '**.njk'
       - '**.scss'
-      - '**.yml'
```

Comment:
did you mean to remove these?

Reply:
I did yeah. I don't think we're using them in the project for anything other than configuring github actions or app engine or some build bot thing. We don't use them for content it seems.